_includes/payment-order-other-features.md

```diff
@@ -596,17 +596,48 @@ Content-Type: application/json
+| {% icon check %} | `paymentorder`                 | `object`     | The payment order object.                                                                                                                                                                                                                                                                |
+| {% icon check %} | `operation`                    | `object`     | `Recur`.                                                                                                                                                                                                                                                                |
+| {% icon check %} | └➔&nbsp;`recurrenceToken`      | `string`     | The created recurrenceToken, if `operation: Verify` or `generateRecurrenceToken: true` was used.                                                                                                                                                                                                                                           |
+| {% icon check %} | └➔&nbsp;`currency`             | `string`     | The currency of the payment order.                                                                                                                                                                                                                                          |
+| {% icon check %} | └➔&nbsp;`amount`               | `integer`    | {% include field-description-amount.md %}                                                                                                                                                                                                                                             |
+| {% icon check %} | └➔&nbsp;`vatAmount`            | `integer`    | {% include field-description-vatamount.md %}                                                                                                                                                                                                                                          |
+| {% icon check %} | └➔&nbsp;`description`          | `string`    | {{% include field-description-description.md documentation_section="paymentorders" %}                                                                                                                                                                                                                                           |
+| {% icon check %} | └─➔&nbsp;`userAgent`           | `string`     | The user agent reference of the consumer's browser - [see user agent definition][user-agent-definition]                                                                                                                                                                                                                                                         |
```

Comment:
Codacy found an issue: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5741678)